python/LAMBDA_INSIDE_VPC/LAMBDA_INSIDE_VPC_test.py

```diff
@@ -0,0 +1,241 @@
+        "vpcConfig": {
+            "subnetIds": [
+                "subnet-123123123",
+                "subnet-123456789"
+            ],
+            "securityGroupIds": [
+                "sg-f236a088"
+            ]
+        }
+    }
+
+    lambda_outside_vpc = {
+        "functionName": "test_function",
+        "functionArn": "arn:aws:lambda:us-west-2:234759432549:function:test_function",
+    }
+
+    lambda_outside_vpc_scenario_2 = {
+        "functionName": "test_function",
+        "functionArn": "arn:aws:lambda:us-west-2:123456789012:function:test_function",
+        "vpcConfig": {
+            "subnetIds": [
+            ],
+            "securityGroupIds": [
+            ]
+        }
+    }
+
+    lambda_different_subnet = {
+        "functionName": "test_function",
+        "functionArn": "arn:aws:lambda:us-west-2:123456789012:function:test_function",
+        "vpcConfig": {
+            "subnetIds": [
+                "subnet-000000000",
+                "subnet-123456789"
+            ],
+            "securityGroupIds": [
+                "sg-f236a088"
+            ]
+        }
+    }
+
+
+    # common scenarios
+    def test_invalid_parameter_value(self):
+        invoking_event = '{"configurationItem":{"configuration":' + json.dumps(self.lambda_inside_vpc) + ',"configurationItemCaptureTime":"2018-07-02T03:37:52.418Z","configurationItemStatus":"ResourceDiscovered","resourceType":"AWS::Lambda::Function","resourceId":"test_function"},"messageType":"ConfigurationItemChangeNotification"}'
+        response = RULE.lambda_handler(build_lambda_configurationchange_event(invoking_event, rule_parameters=self.rule_invalid_parameter), {})
+        assert_customer_error_response(self, response, 'InvalidParameterValueException', 'Invalid value for paramter "subnetId", Expected subnet Id')
+
+    # Scenario 2
+    def test_empty_parameter_value(self):
+        invoking_event = '{"configurationItem":{"configuration":' + json.dumps(self.lambda_inside_vpc) + ',"configurationItemCaptureTime":"2018-07-02T03:37:52.418Z","configurationItemStatus":"ResourceDiscovered","resourceType":"AWS::Lambda::Function","resourceId":"test_function"},"messageType":"ConfigurationItemChangeNotification"}'
+        response = RULE.lambda_handler(build_lambda_configurationchange_event(invoking_event, rule_parameters=self.rule_emptry_parameter_value), {})
+        resp_expected = []
+        resp_expected.append(build_expected_response('COMPLIANT', 'test_function', 'AWS::Lambda::Function'))
+        assert_successful_evaluation(self, response, resp_expected)
+
+    # Scenario 3.a
+    def test_lambda_outside_vpc(self):
+        invoking_event = '{"configurationItem":{"configuration":' + json.dumps(self.lambda_outside_vpc) + ',"configurationItemCaptureTime":"2018-07-02T03:37:52.418Z","configurationItemStatus":"ResourceDiscovered","resourceType":"AWS::Lambda::Function","resourceId":"test_function"},"messageType":"ConfigurationItemChangeNotification"}'
```

Comment:
Can you create a function that does that. You do a lot of times the same thing in all scenarios :)